valpas-web/src/views/Raamit.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from "react"
+import { Route } from "react-router-dom"
+import { CurrentUser, isLoggedIn } from "../state/auth"
+
+const runningLocally = window.environment == "local"
+const localRaamitEnabled = runningLocally && !process.env.VIRKAILIJA_RAAMIT_HOST
```

Comment:
tämä ei toimi halutulla tavalla, eli että voisi ajaa valpasta tämän parametrin avulla omalla koneella ja näytettäisi tuotanto/untuva/qa-raamit

Reply:
johtuu siitä, että process.env kait luetaan jo ts-käännöksessä, bundlauksessa tms.

Reply:
ihan ok, jos tarvii bundlata uudestaan, mutta sitten voisi package.jsoniin laittaa komennot, jotka tekee sen oikein

Reply:
hmm, komento takaisin, toimiihan tuo, ei vaan toimi login-ikkunassa

Reply:
sori, ei toimikaan ainakaan vakaasti, kun vaihtelee start:n ja start:raamit välillä ja reloadailee selainta

Reply:
Alkaa muuten toimimaan kun `start:raamit` komennon jälkeen tekee koodiin minkä tahansa kääntämiseen johtavan muutoksen 🙈 Mutta ei tosiaan suoraan käynnistyessä.

Onko tää sellainen keissi jonka korjaamiseen kannattaa käyttää vielä aikaa? Katoin tätä ja suoraan sanoen en ymmärrä yhtään miksi parcel toimii näin. Oispa webpack.